lib/services/ios-project-service.ts

```diff
@@ -350,12 +351,92 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 
 	public prepareAppResources(appResourcesDirectoryPath: string): IFuture<void> {
 		return (() => {
+			let platformFolder = path.join(appResourcesDirectoryPath, this.platformData.normalizedPlatformName);
+			let filterFile = (filename: string) => this.$fs.deleteFile(path.join(platformFolder, filename)).wait();
+
+			// These got copied but are not actually resources. They should be rather filterd before the copy in the first place but anyway, let's just delete them here.
+			filterFile("Info.plist");
+			// filterFile("Podfile");
+			// filterFile("Entitlements.plist");
+
 			this.$fs.deleteDirectory(this.getAppResourcesDestinationDirectoryPath().wait()).wait();
 		}).future<void>()();
 	}
 
 	public processConfigurationFilesFromAppResources(): IFuture<void> {
-		return Future.fromResult();
+		return (() => {
+			this.mergeInfoPlists().wait();
+		}).future<void>()();
+	}
+
+	private mergeInfoPlists(): IFuture<void> {
+		return (() => {
+			let projectDir = this.$projectData.projectDir;
+			let infoPlistPath = path.join(projectDir, constants.APP_FOLDER_NAME, constants.APP_RESOURCES_FOLDER_NAME, this.platformData.normalizedPlatformName, "Info.plist");
+
+			if (!this.$fs.exists(infoPlistPath).wait()) {
+				// The project is missing Info.plist, try to populate it from the project tempalte.
+				let projectTemplateService = (<IProjectTemplatesService>this.$injector.resolve("projectTemplatesService"));
+				let defaultTemplatePath = projectTemplateService.defaultTemplatePath.wait();
+				let templateInfoPlist = path.join(defaultTemplatePath, constants.APP_RESOURCES_FOLDER_NAME, "iOS", "Info.plist");
```

Comment:
instead of "iOS", you can use 
```
this.$devicePlatformsConstants.iOS
```
Instead of `Info.plist` you can use:
```TypeScript
this.platformData.configurationFileName
```
Sooo, the code could be like this:
```TypeScript
let templateInfoPlist = path.join(defaultTemplatePath, constants.APP_RESOURCES_FOLDER_NAME, this.$devicePlatformsConstants.iOS, this.platformData.configurationFileName);
```
> NOTE: You'll have to add `devicePlatformsConstants` to the constructor: 
```TypeScript
private $devicePlatformsConstants: Mobile.IDevicePlatformsConstants
```